components/BuildingMap/BuildingMap.jsx

```diff
@@ -0,0 +1,207 @@
+import React from 'react';
+import
+{
+    HOST_URL
+}
+from '../../yasp.config';
+import
+{
+    pad
+}
+from '../../util/utility';
+const buildingData = [
+    {
```

Comment:
can we move this buildingData to its own file as a module and import it so our component doesn't have to have that living in it directly?